08-message-translations.Rmd

```diff
@@ -110,50 +148,126 @@ msgstr[2] "Įspėjančių pranešimų:\n"
+`.po` files are plain text, but while helpful for human readers, this is
+inefficient for consumption by computers. The .mo format is a "compiled"
+version of the .po file optimized for retrieving messages when R is
+running.
 
-In R-devel, the conversion from .po to .mo is done by R Core -- you don't need to compile these files yourself.
-They are stored in the R sources at `./src/library/translations/inst` in various language-specific subdirectories.
+In R-devel, the conversion from .po to .mo is done by R Core -- you
+don't need to compile these files yourself. They are stored in the R
+sources at `./src/library/translations/inst` in various
+language-specific subdirectories.
 
 ## How to contribute new translations
 
 <!-- TODO(michaelchirico): Creating and editing .po files, testing the translations worked, **encoding**, translation teams, release schedule -->
 
+Translating R into different languages helps make it more user-friendly
+for non-English speakers and grow the R community. Also, read the blog
+how [R can use your help: Translating R
+message](https://blog.r-project.org/2022/07/25/r-can-use-your-help-translating-r-messages).
+
+To get started with new translation, please follow the steps below:
+
+Step 1: Register an Account at Weblate
+
+Weblate is an open-source platform for collaborative translation of
+software projects, including R. Register an account at
+<https://translate.rx.studio/> to start contributing.
+
+Step 2: Choose a Component and Language
+
+Select a component of R with less than 100% translation. You can start
+with sections like utils, base (R files), or tools (R files). Then you
+can select your preferred language.
+
+![](img/translate_component.png)
+
+Step 3: Check String Status
+
+Click on "Unfinished strings" to see the strings of messages that need
+translation.
+
+Here:
+
+-   Translated Strings: Already translated.
+-   Review Pending Strings: Translated strings awaiting review.
+-   Untranslated/Incomplete Strings: Strings requiring translation.
+
+![](img/unfinished_strings_translate.png)
+
+Step 4: Translate the Message
+
+Translate the message by typing the translation in the text box.
+
+![](img/translate_string_and_save.png)
+
+Click "Save and Continue" to save the translation and continue. The
+translation team will review and integrate it into the R interface if it
+meets the quality standards.
+
+- Note: If you're unsure about the translate, please check needs editing. This will ensure other reviewer will check and verify if your translations are correct.
+
+
+Some Tips to follow:
+
+- Be consistent: Use the same words and phrases throughout the translation to make it consistent and avoid confusion.
+
+- Use concise language: This will ensure the translated messages are short and easy to read
+
+- Review the translations: After finishing the translation, check if you have any alerts or warning in the weblate string status.
```

Comment:
```suggestion
- Check for technical issues: After finishing the translation, check if you have any alerts or warning in the weblate string status, e.g. double instead of single space.
```